master/getting-started/kubernetes/installation/manifests/allow-istio-pilot.yaml

```diff
@@ -0,0 +1,19 @@
+# This NetworkPolicy allows pods egress access the to 
```

Comment:
Any reason this is in `manifests/` instead of `manifests/app-layer-policy`?

Same for the other few manifests here...

Reply:
Good idea